### PR TITLE
Add basics/instructions translation

### PR DESCRIPTION
**Pull request by aliaksandr-s**

### CODE REVIEW

**Comment by aliaksandr-s** [comment]
@freetonik Here is one more)

File: modules/10-basics/40-instructions/description.en.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+
+name: Statements
+theory: |
+  Statement is an instruction for a computer. PHP code is a set of statements separated by `;`.
+
+  Here is the example of the code with two statements. When you run this code you will see two sentences on the screen, one after the other.
```
Comment (freetonik) [inline modules/10-basics/40-instructions/description.en.yml]: an example of code

**Comment by aliaksandr-s** [comment]
@freetonik updated

**Comment by freetonik** [comment]
Oh yeah! Thanks!

**Pull request merged**